state/state_test.go

```diff
@@ -1466,20 +1466,24 @@ func (s *StateSuite) TestMachineCountForSeries(c *gc.C) {
 		"win2008r2", "win2012", "win2012", "win2012hv", "win2012hvr2", "win2012r2", "win2012r2",
```

Comment:
seems this change lost https://github.com/juju/juju/blob/2.9/state/state_test.go#L1467

Reply:
we need to define the windows series manually since 3.0 has deleted them from core